proposals/0522-or-patterns.rst

```diff
@@ -74,49 +74,80 @@ Proposed Change Specification
 Changes in the grammar
 ~~~~~~~~~~~~~~~~~~~~~~
 
+We consider this as an extension to GHC's .
 We consider this as an extension to `Haskell 2010 grammar
```

Comment:
Implementation of the proposal should not disrupt the processing of modules where the extension is not active, but implementing it by adjusting `Parser.y` in the way you are suggesting would be disruptive wouldn't it? Could you please clarify.